packages/cli/src/controllers/auth.controller.ts

```diff
@@ -44,14 +45,19 @@ export class AuthController {
 		res: Response,
 		@Body payload: LoginRequestDto,
 	): Promise<PublicUser | undefined> {
-		const { email, password, mfaCode, mfaRecoveryCode } = payload;
+		const { emailOrLdapUsername, password, mfaCode, mfaRecoveryCode } = payload;
 
 		let user: User | undefined;
 
 		let usedAuthenticationMethod = getCurrentAuthenticationMethod();
+
+		if (usedAuthenticationMethod === 'email' && !isEmail(emailOrLdapUsername)) {
+			throw new BadRequestError('Invalid email address');
```

Comment:
Should we add a test to check that the expected `BadRequestError` is thrown in the `auth.controller.test.ts`? Or are we just catching this in the integration tests?

Reply:
Yes, we have a test for that here -> https://github.com/n8n-io/n8n/pull/13605/files#diff-301108761b1af4c984d648c562eddfc5dd57bde2b5fce077fe23d2c46cacfd07R151